app/pages/Home/components/GroupsList.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Group } from '~/features/dataProvider'
+import { Link } from '@remix-run/react'
+
+export type GroupsListProps = {
+  groups: Group[]
+}
+
+export const GroupsList: React.FC<GroupsListProps> = ({ groups }) => {
+  return (
+    <section className="py-16 grid gap-8">
+      <div>
+        <Link to="/groups" className="hover:underline">
+          <h2 className="text-3xl font-extrabold text-gray-900 sm:text-4xl mb-12">
+            Participating Member Groups
+          </h2>
+        </Link>
+        <div className="grid gap-8 grid-cols-1 md:grid-cols-2 lg:grid-cols-4 items-center">
+          {groups.map((group) => (
+            <div
+              key={group.name}
+              className="w-full p-1 flex justify-center block rounded-lg border border-gray-200 shadow-md"
+            >
+              <img className="h-32" src={group.logo.png} alt={group.name} />
```

Comment:
We have the size for each image, maybe we can set that here to keep the aspect ratio? Some of the images look distorted.

Reply:
If I use the width and height available from the json, I will end up with images of different sizes and ratios. 

<img width="1289" alt="Screenshot 2022-09-14 at 17 54 18" src="https://user-images.githubusercontent.com/32243459/190217611-63278f9c-568b-40e7-9893-e4556e452c4c.png">

In order to keep the ratio between the width and height and avoid distortion, I think we need to do something like this:
```jsx
<img className="h-32 object-cover w-full" />
```

[object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit#values)

Reply:
maybe we can use a "card" and fit the image in there, like it's usually done on logo panels etc.

![Screenshot 2022-09-14 at 9 38 45 PM](https://user-images.githubusercontent.com/74390/190235756-3d9dba22-e9de-4733-b474-4b4f4f2d5edc.png)

Reply:
https://remix-image.mcfarl.in/docs/component

Reply:
I need to check the component

Reply:
I just pushed a commit which uses the remix-image component, but still, I can't set explicitly the width and height of the image. However, the remix-image component ensures that the ratio of the image is constant. 

For example, the initial size of the below image is 525x525 with a ratio of 1:1. We render it with a size of 225x225, which is the same ratio.

<img width="901" alt="Screenshot 2022-09-18 at 00 30 41" src="https://user-images.githubusercontent.com/32243459/190879713-c48ed99e-5e02-40c3-a36b-4588cb44c216.png">